app/services/courses/fetch.rb

```diff
@@ -1,8 +1,8 @@
 module Courses
   class Fetch
     class << self
```

Comment:
I'm not too sure of the usefulness of this `service` maybe we should look at removing it altogether @darokel

Reply:
Yup, we should get rid of it if we can. I think it was used as a way to deal with the lookup soup in old publish

Reply:
Yeah lets card that up

Reply:
It's now in the dev backlog